app/routes/cms/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { Center, Heading } from "@chakra-ui/react";
+import { json, LoaderFunction, useLoaderData } from "remix";
+import { requireUserId } from "~app/auth";
+
+export const loader: LoaderFunction = async ({ request }) => {
+	return json({ userId: await requireUserId(request) });
+};
+
+interface LoaderData {
+	userId: number;
+}
+
+export default function Index() {
+	const { userId } = useLoaderData<LoaderData>();
+
+	return (
+		<Center h="100vh">
```

Comment:
On a small viewport, this will cause content clipping, instead of making it horizontally scrollable